tools/selection_tools.cpp

```diff
@@ -194,17 +194,17 @@ static void SelectUnassignedElementsHelper(
 
 template <class TGeomObj>
 static void DeselectUnassignedElementsHelper(
```

Comment:
Do you actually still use this method? If not I would rather remove it.

Reply:
You are right, I do not use it anymore, it's just left over from the previous implementation. Shall I remove it or will you do it?

Reply:
Would be great if you could do it. Thanks!